pandas/io/tests/test_json/test_pandas.py

```diff
@@ -324,12 +324,12 @@ def test_frame_to_json_except(self):
     def test_frame_empty(self):
         df = DataFrame(columns=['jim', 'joe'])
         self.assertFalse(df._is_mixed_type)
-        assert_frame_equal(read_json(df.to_json()), df)
```

Comment:
I believe `df.to_json() == '{"jim":{},"joe":{}}'`. It *is* checking that the number / names of columns is correct.